lib/modules/syntaxHighlighting.js

```diff
@@ -0,0 +1,20 @@
+/* @flow */
+
+import hljs from 'highlight.js';
+import { Module } from '../core/module';
+import { watchForThings } from '../utils';
+
+export const module: Module<*> = new Module('syntaxHighlighting');
+
+module.moduleName = 'syntaxHighlightingName';
+module.category = 'appearanceCategory';
+module.description = 'syntaxHighlightingDesc';
+module.disabledByDefault = false;
+
+module.beforeLoad = () => {
+	watchForThings(['post', 'comment', 'message'], highlight);
```

Comment:
Doesn't seem like Highlight.js wants to take in a HTMLCollection element in this way from what I've seen. Not sure if it would work with the whole element since I tested it with jQuery and selecting the closest pre tag.